docs/operations-guide/upgrading-metabase.md

```diff
@@ -1,31 +1,60 @@
 # Upgrading Metabase
 
-Before you attempt to upgrade Metabase, you should make a backup of the application database just in case. While it is unlikely you will need to roll back, it will do wonders for your peace of mind.
+## Step 1: Back up your application database
 
-How you upgrade Metabase depends on how you are running it. See below for information on how to update Metabase on managed platforms.
+The application database keeps track of all of your people, dashboards, questions, collections, permissions: all the application data in Metabase (that is, everything but the data you've connected to Metabase). While it's unlikely you'll need to roll back to your current version, a backup will do wonders for your peace of mind.
 
-## Specific Platforms
+See [Backing up Metabase application data](backing-up-metabase-application-data.md).
 
-### Docker Image
+## Step 2: Swap in the new Metabase version
 
-If you are running Metabase via docker, then you simply need to stop the Docker container and start a new container with the latest Metabase image. On startup, Metabase will perform any upgrade tasks it needs to perform, and once it's finished you'll be running the new version.
+Steps differ depending on whether you're running the JAR or a Docker image.
 
-To pull the latest Metabase:
+- [Upgrading the JAR](#upgrading-the-jar)
+- [Upgrading the Docker image](#upgrading-the-docker-image)
 
-    $ docker pull metabase/metabase:latest
+### Upgrading the JAR
 
-### Jar file
+If you're running the JVM Jar file directly:
 
-If you are running the JVM Jar file directly, then you simply kill the process, replace the .jar file with the newer version and restart the server. On startup, Metabase will perform any upgrade tasks it needs to perform, and once it's finished you'll be running the new version.
+1. [Backup your application database](backing-up-metabase-application-data.md).
```

Comment:
It may be helpful to add some of those warnings/links to migrating docs to **backing-up-metabase-application-data.md** as well - it talks about using Metabase in a production environment and then the first section is about backing up the default H2 database.